website/content/docs/upgrade/upgrade-specific.mdx

```diff
@@ -62,6 +62,18 @@ from the Nomad client by setting [`set_environment_variables`][artifact_env].
 The use of filesystem isolation can be disabled in Client configuration by
 setting [`disable_filesystem_isolation`][artifact_fs_isolation].
 
+#### Server `rejoin_after_leave` (default: `false`) now enforced
+
+All Nomad versions prior to v1.4.4 have incorrectly ignored the Server [`rejoin_after_leave`][rejoin_after_leave]
```

Comment:
This link is broken. See the preview at https://nomad-a6lg3nb9m-hashicorp.vercel.app/nomad/docs/upgrade/upgrade-specific#server-rejoin_after_leave-default-false-now-enforced.

The best way to fix this is to have this line be:

```suggestion
All Nomad versions prior to v1.4.4 have incorrectly ignored the Server [`rejoin_after_leave`][]
```

And then add the following line to the foot of this file:

```
[`rejoin_after_leave`]: /docs/configuration/server#rejoin_after_leave
```